plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -149,7 +149,7 @@
          on to the appropriate Host (virtual host).
          Documentation at /docs/config/engine.html -->
 
-    <Engine name="Catalina" defaultHost="localhost">
+    <Engine name="Catalina" defaultHost="localhost" jvmRoute="${exo.cluster.node.name}">
```

Comment:
This is not mandatory to realize load balancing with sticky session. 
IMO, we should not impose this to the users as it can be configure via environment variables.
The admins could want to use short names for jvmRoute and longer names for node names in the cluster. Also the load balancers mostly use their own cookies to identify the nodes in the cluster.